hcs-sxc-java-examples/hcs-sxc-java-settlement-demo/config/docker-compose.sample.yml

```diff
@@ -38,6 +51,7 @@ services:
      COLOR: 275f4f
    ports:
      - 8083:8083
+>>>>>>> 3eb863335fb668ddbf1ae6e9f9e0b7a1261d4483
```

Comment:
Merge gone wrong ?

Reply:
I had disabled the examples. merging bits only until things work.